src/Store/KeyBindingsStoreConnector.re

```diff
@@ -202,7 +204,7 @@ let start = () => {
           | Some(fileName) => fileName
           };
         if (bufferFileName == configPath && cmd == Vim.Types.BufWritePost) {
-          Oni_Core.Log.info("Reloading key bindings from: " ++ configPath);
+          Log.info("Reloading key bindings from: " ++ configPath);
           dispatch(Actions.KeyBindingsReload);
```

Comment:
Kept despite the dispatched action because the log message includes additional information